README.md

```diff
@@ -13,34 +13,12 @@ A collection of code generation and libraries to for API development.
 See `test/mock_resources.proto` and `test/generate.go` for an example of how to use solo-kit
 
 ## Build
-- clone repo to gopath 
-- gather dependencies: `dep ensure -v`
+- clone repo locally
+- gather dependencies: `go mod download`
 - use binary produced with `make solo-kit-gen` or import `cmd.Run(...)` into your own code gen code 
 
 ## Usage
 - re-run whenever you change or add an api (.proto file)
-- api objects generated from messages defined in protobuf files with magic comments prefixed with `@solo-kit`
+- api objects generated from messages defined in protobuf files which have 
```

Comment:
good idea updating this, but comment is unfinished

Reply:
I will be doing a full once over of the docs, as well as updating for everything new